### PR TITLE
New Nodes

### PR DESCRIPTION
**Pull request by darkdrgn2k**
E-Mesh, Corrected Directional,  Added GW, and proposed directional\
Added new WestMount node
Added dasanchez Node
Added Datacenter node

### CODE REVIEW

**Comment by darkdrgn2k** [comment]
Data center nodes needed to be added as most of the nodes will be peering with them.  Without them can't create on-map links.

**Comment by darkdrgn2k** [comment]
Question:

How do you want to handle Directional Potential nodes.

IE: 
All the potential nodes right now are directional to indicate which way the unit is facing.
New nodes are being added as OMNI directional because that is what they are

Do we want to add an additional Potential node where there is a potential for a directional link? (see proposed node in this commit) or just leave it as omni?

File: nodeList.json [hunk 1]
```diff
@@ -759,16 +759,39 @@
...
+      "name":"E-Mesh Balcony",
+      "latitude":43.68306736430572,
+      "longitude":-79.52565314047547,
+      "cardinalDirection":"North",
+      "floor":"13",
+      "status":"proposed",
+      "dateAdded":"2017-08-15T00:00:00.0Z"
+   },
+   {
+      "name":"benhylau",
```
Comment (garrying) [inline nodeList.json]: This line to `794` is reverting #11.

File: nodeList.json [hunk 2]
```diff
@@ -790,13 +813,23 @@
       "name": "E-Mesh North York GW",
       "latitude": 43.77966238349766,
       "longitude":-79.34211684973752,
-      "cardinalDirection": "West",
+      "cardinalDirection": "Omni",
       "status":"active",
       "floor": 3,
       "IPV6Address":"north-york-gw.darkdrgn2k.h.e-mesh.net",
```
Comment (garrying) [inline nodeList.json]: Can we have the IPv6 address here? Context: There's a missing IPv6 format prop in the schema that will be added. This value will fail the spec.

Comment (benhylau) [reply nodeList.json]: Agreed, IP only in this field pls

Comment (darkdrgn2k) [reply nodeList.json]: Host Names would have reduced the number of PRs required when nodes are rebuilt. 
However moved to IPv6 addy.

**Pull request merged**